setup.py

```diff
@@ -76,6 +76,8 @@
         'Programming Language :: Python :: 3.5',
```

Comment:
Please remove 3.4 and 3.5

Reply:
The readme states that python 3.4 is supported, so if any the CI should also contain 3.4 and 3.5, right?
Or the readme is wrong and it works only on 3.6+?

Reply:
Please fix README.md as well. We do not support EOL'ed versions.

Reply:
done